mapclientplugins/datamapstep/model/mappingmodel.py

```diff
@@ -79,29 +79,25 @@ def _autorange_spectrum(self):
         spectrum.autorange(scene, scene_filter)
 
     def create_graphics(self):
-        model_coordinates = self._mapper.get_model_coordinate_field()
-        data_coordinate = self._mapper.get_data_coordinate_field()
         with ChangeManager(self._scene):
             self._scene.removeAllGraphics()
+
             data_points = self._scene.createGraphicsPoints()
             data_points.setFieldDomainType(Field.DOMAIN_TYPE_DATAPOINTS)
-            data_points.setCoordinateField(data_coordinate)
             point_attr = data_points.getGraphicspointattributes()
-            point_attr.setBaseSize([10, 10, 10])
+            point_attr.setBaseSize([200, 200, 200])
```

Comment:
This may need to become a configurable item in the user interface, it is related to the size of the mesh.  If someone has a mesh that is very small then these points are going to be very big in comparison.